modules/apps/site-initializer/site-initializer-liferaybotics/site-initializer-liferaybotics-test/src/testFunctional/tests/Liferaybotics.testcase

```diff
@@ -96,4 +99,98 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-150999. Verify all the links present in the Dashboard Page related with News redirect to their respective pages"
+	@ignore = "true"
+	@priority = "4"
+	test VerifyTheLinksRedirectsToItsPage {
+		task ("Go to Liferaybotics Site") {
+			ApplicationsMenu.gotoSite(site = "${testSiteName}");
+		}
+
+		task ("Go to  Dashboard Page on Liferaybotics Site") {
+			Click(
+				labelOption = "Dashboard",
+				locator1 = "Liferaybotics#MENU_LABEL_NAME");
+		}
+
+		task ("Click in The 'All New Series of TX Robotic Machine' link and assert it redirects to Blog Page") {
+			Click(
+				key_title = "${testNewsArticleName1}",
+				locator1 = "Liferaybotics#NEWS_LINK");
+		}
+
+		task ("Assert the Article Page is present") {
+			AssertElementPresent(
+				key_title = "${testNewsArticleName1}",
+				locator1 = "Liferaybotics#NEWS_TITLE");
+		}
+
+		task ("Click in Back icon to return to Dashboard Page") {
+			Click(locator1 = "Liferaybotics#NEWS_BACK_ICON");
+		}
+
+		task ("Click in The 'Easier to operate than never before' link and assert it redirects to Blog Page") {
+			Click(
+				key_title = "${testNewsArticleName2}",
+				locator1 = "Liferaybotics#NEWS_LINK");
+		}
+
+		task ("Assert the Article Page is present") {
+			AssertElementPresent(
+				key_title = "${testNewsArticleName2}",
+				locator1 = "Liferaybotics#NEWS_TITLE");
+		}
+
+		task ("Click in Back icon to return to Dashboard Page") {
+			Click(locator1 = "Liferaybotics#NEWS_BACK_ICON");
+		}
+
+		task ("Click in The 'Grab the best deals this June' link and assert it redirects to Blog Page") {
+			Click(
+				key_title = "${testNewsArticleName3}",
+				locator1 = "Liferaybotics#NEWS_LINK");
+		}
+
+		task ("Assert the Article Page is present") {
+			AssertElementPresent(
+				key_title = "${testNewsArticleName3}",
+				locator1 = "Liferaybotics#NEWS_TITLE");
+		}
+
+		task ("Click in Back icon to return to Dashboard Page") {
+			Click(locator1 = "Liferaybotics#NEWS_BACK_ICON");
+		}
+	}
+
+	@description = "This is a test for LPS-150999. Verify Product Publisher contained a Warranty product"
+	@ignore = "true"
+	@priority = "4"
+	test VerifyTheProductPublisherContainsWarrantyProduct {
+		task ("Go to Liferaybotics Site") {
+			ApplicationsMenu.gotoSite(site = "${testSiteName}");
+		}
+
+		task ("Go to  Dashboard Page on Liferaybotics Site") {
+			Click(
+				labelOption = "Dashboard",
+				locator1 = "Liferaybotics#MENU_LABEL_NAME");
+		}
+
+		task ("Assert the Latest Products title is present") {
+			AssertElementPresent(
+				key_title = "Latest Products",
+				locator1 = "Liferaybotics#DASHBOARD_TITLE");
+		}
+
+		task ("Assert Product Publisher widget is present") {
+			AssertElementPresent(locator1 = "Liferaybotics#PRODUCT_PUBLISHER");
+		}
+
+		// TO-DO:Update when Warranty Page is working
```

Comment:
Do we have a ticket for this? Did we add a note in that ticket to don't forget this TO-DO?

Reply:
Ticket for this update is: https://issues.liferay.com/browse/LPS-152264